ui/app/components/clients/dashboard.js

```diff
@@ -101,6 +101,17 @@ export default class Dashboard extends Component {
   get responseTimestamp() {
     return this.args.model.activity?.responseTimestamp;
   }
+
+  get upgradeDate() {
+    let keyInfoObject = this.args.model.versionHistory.keyInfo;
```

Comment:
If the version history is not defined we should add a check here or we'll get a `versionHistory is undefined` error
```suggestion
    let keyInfoObject = this.args.model.versionHistory?.keyInfo;
```